contrib_bots/lib/CommuteBot/CommuteBot.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import print_function
+import requests
+
+class CommuteHandler(object):
+    '''
+    This plugin provides information regarding commuting from an origin to a destination, providing
+    a multitude of information. It looks for messages starting with '@commute'.
+    '''
+
+    def usage(self):
+        return '''
+            This plugin will allow briefings of estimated travel times,
+              distances and fare information for transit travel. It can vary outputs
+              depending on traffic conditions, departure and arrival times as well as
+              user preferences (toll avoidance, preference for bus travel, etc.).
+            It looks for messages starting with '@commute'.
+
+            Users should input an origin, destination and a mode of transport.
+            '''
```

Comment:
This description doesn't help with knowing how to actually input origin, destination, and mode of transport. Should I write `@commute Warsaw New York bus`?

This description is the first thing you see when running the bot, without looking into its code. Could you make sure it's clear here how to actually run the bot?